lib/book.rb

```diff
@@ -1,2 +1,17 @@
+require 'pry'
 
+class Book
+  attr_accessor :title, :author, :page_count, :genre
 
+  def initialize(title, author = nil, page_count = nil, genre = nil )
```

Comment:
Like we talked about this morning, don't use a default value unless you _want_ a default value.